src/codeu/chat/client/ClientConversation.java

```diff
@@ -140,13 +142,19 @@ public void startConversation(String title, Uuid owner) {
 	System.out.println(this.getClass().toString() + " startConversation()");
     final boolean validInputs = isValidTitle(title);
 
-    final Conversation conv = (validInputs) ? controller.newConversation(title, owner) : null;
+    final Conversation conv = (validInputs) ? view.newConversation(title, owner) : null;
 
     if (conv == null) {
       System.out.format("Error: conversation not created - %s.\n",
           (validInputs) ? "server failure" : "bad input value");
+      return;
     } else {
       LOG.info("New conversation: Title= \"%s\" UUID= %s", conv.title, conv.id);
+      
+      summariesByUuid.put(conv.summary.id, conv.summary);
```

Comment:
Comments on each of these explaining why you need each index would be good. JavaDoc on the variables would be enough.